### PR TITLE
[[ Community Docs ]] IS A operator fixes

### PR DESCRIPTION
**Pull request by asayd**
- Fixed formatting of list of value types;
- Reference value (glossary) instead of value (function);
- Reviewed and fixed links and references.

### CODE REVIEW

File: docs/dictionary/operator/is-a.lcdoc [hunk 1]
```diff
@@ -35,42 +35,44 @@ Example:
...
-
-        * integer if it consists of digits (with an optional leading minus sign), or an hexadecimal number (such as 0x0F9B)
-
-        * number if it consists of digits, optional leading minus sign, optional decimal point, and optional "E" or       "e" (scientific notation), or an optional leading minus sign and infinity or inf
-
-        * point if it consists of two numbers separated by a comma
-
-        * rect if it consists of four numbers separated by commas
-        * ASCII string if it does not contain any characters greater than charToNum(127)
+* <array(keyword> if its <keys> function is not empty.
```
Comment (livecodeali) [inline docs/dictionary/operator/is-a.lcdoc]: There's a missing closing parenthesis here

**Comment by livecodeali** [comment]
Currently the `value` parameter is conflicting with the link to the glossary entry for value. I wonder if the parameter name should be changed to something like `valueToCheck` here and in the syntax definition.

OTOH there are some things (value, container, object, control, textChunk, fieldChunk, etc...) which are sort of generic syntax elements, so maybe we should be doing something slightly different with these.

Perhaps this needs further thought.

File: docs/dictionary/operator/is-a.lcdoc (hunk 1 above)
Comment (asayd) [inline docs/dictionary/operator/is-a.lcdoc]: Fixed and submitted.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok bb7530c

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok bb7530c87e0a00c3537fb985d18ac61e4cecafea

**Comment by asayd** [comment]
@livecodeali I see what you mean RE `value`. On the one hand its an appropriate placeholder, but you'd like to be able to link it to the glossary entry. It seems that under your markdown scheme angle bracket notation is overloaded: in the syntax field it means "make italic", but in the rest of the fields it means "link to another entry". Maybe the solution is to use a different way of indicating italic text.

**Comment by livecode-vulcan** [comment]
😎 test success bb7530c87e0a00c3537fb985d18ac61e4cecafea
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2434)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2444)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2427)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2445)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2451)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2455)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2467)

**Pull request merged**